src/polytope_blmos.jl

```diff
@@ -93,6 +93,10 @@ function rounding_hyperplane_heuristic(tree::Bonobo.BnBTree, tlmo::TimeTrackingL
     for idx in tree.branching_indices
         z[idx] = round(x[idx])
     end
+
+    if sum(iszero.(z[tree.branching_indices])) == length(z[tree.branching_indices])
```

Comment:
```suggestion
    if count(!iszero, z[tree.branching_indices]) == 0
```
avoids multiple allocations and indexing